settings.gradle

```diff
@@ -13,19 +13,13 @@ if (hasProperty("CORE_VISION_AR")) {
     project(':CoreAr').projectDir = new File(CORE_VISION_AR)
 }
 
-if (hasProperty("VISION_SDK")) {
```

Comment:
It will fail for those who don't have this properties now. Think better way is to define those properties in `.gradle/gradle.properties`